pkg/common/util/webhooks.go

```diff
@@ -0,0 +1,25 @@
+package util
+
+import (
+	v1 "github.com/kubeflow/training-operator/pkg/apis/kubeflow.org/v1"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/validation"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+var SupportedJobControllers = sets.New(
+	v1.MultiKueueController,
+	v1.KubeflowJobsController)
+
+func ValidateManagedBy(runPolicy *v1.RunPolicy, allErrs field.ErrorList) field.ErrorList {
+	if runPolicy.ManagedBy != nil {
+		manager := *runPolicy.ManagedBy
+		fieldPath := field.NewPath("spec", "managedBy")
+		allErrs = append(allErrs, validation.IsDomainPrefixedPath(fieldPath, manager)...)
+		if !SupportedJobControllers.Has(manager) {
+			allErrs = append(allErrs, field.NotSupported(fieldPath, manager, sets.List(SupportedJobControllers)))
```

Comment:
```suggestion
			allErrs = append(allErrs, field.NotSupported(fieldPath, manager, SupportedJobControllers.UnsortedList()))
```

Do we need an ordered list here?

Reply:
I think the main motivation to return such lists ordered in the core k8s is to make sure that the output is deterministic ([example](https://github.com/kubernetes/kubernetes/blob/746f08a8da0a45e8b1a46362822db05cb11ed294/pkg/apis/batch/validation/validation.go#L334)). This is helpful when writing tests.